quill-core/src/test/scala/io/getquill/quotation/QuotationSpec.scala

```diff
@@ -857,7 +859,7 @@ class QuotationSpec extends Spec {
       }
       "abritrary" in {
         val q = quote(lift(String.valueOf(1)))
-        q.liftings.`java.this.lang.String.valueOf(1)`.value mustEqual String.valueOf(1)
+        //q.liftings.`java.this.lang.String.valueOf(1)`.value mustEqual String.valueOf(1)
```

Comment:
seems like my previous message has been lost with rebase. I'm not sure what should I do here. `this` must be present in 2.11, but in 2.12 should be removed